### PR TITLE
[MC] Swap Daytona w/ MerriweatherSans + use variable fonts

### PR DESCRIPTION
**Pull request by morgaan**
* BREAKING:
    * Switch springernature context fonts to Variable fonts
    * Swap Daytona with Merriweather Sans
    * Remove all springernature context fonts from the fonts folder
* Add the 2 new variable fonts into springernature context fonts folder
* Add documentation about fonts in springernature context README

### CODE REVIEW

File: context/brand-context/springernature/README.md [hunk 1]
```diff
@@ -1,3 +1,89 @@
+## Fonts
+
+We are using Merriweather as a variable font for both `serif` and `sans-serif`.
+
+Browsers who support variable fonts do support the `woff2` format. We therefore
```
Comment (alexkilgour) [inline context/brand-context/springernature/README.md:5]: "Browsers that support variable fonts also support the `woff2` format

File: context/brand-context/springernature/README.md [hunk 2]
```diff
@@ -1,3 +1,89 @@
...
+do not need to serve other formats.
+
+Browsers who do not support variable fonts, will fallback to the system fonts
+specified in the `$context--font-family-serif` and `$context--font-family-sans`
+SASS variables after respectively `'Merriweather'` and `'Merriweather Sans'`.
```
Comment (alexkilgour) [inline context/brand-context/springernature/README.md:10]: SASS variable, respectively

(remove after, replace with comma)

File: context/brand-context/springernature/README.md [hunk 3]
```diff
@@ -1,3 +1,89 @@
...
+
+So far our `Merriweather` font instance only includes the [Weight variation
+axis](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Fonts/Variable_Fonts_Guide#weight).  
+At the time of writing `Merriweather` supports the additional [Width
+variation](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Fonts/Variable_Fonts_Guide#width) and [Optical Size variation](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Fonts/Variable_Fonts_Guide#optical_size) as well.  
+If at any point in the future there is a need for these variations, you will
+need to start over the font generation process as described below, leaving in
+the variations you need. **Beware** though, the font file weight will increase
+with the amount of variations you add.
+At the time of writeing `Merriweather` only supports the [Weight variation
```
Comment (alexkilgour) [inline context/brand-context/springernature/README.md:20]: writeing -> writing

File: context/brand-context/springernature/README.md [hunk 4]
```diff
@@ -1,3 +1,89 @@
...
+axis](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Fonts/Variable_Fonts_Guide#weight).  
+It is worth noting that we also optimised the available `font-weight` range at
+the font generation step to only span from `300` to `700`.  
+If at any point in the future there is a need for a different range, you will
+need to start over the font generation process as described below, specifying
+the new range. **Beware** though, the font file weight will increase the broader
+the range is.
+
```
Comment (alexkilgour) [inline context/brand-context/springernature/README.md:28]: You start by saying "At the time of writing `Merriweather` supports the additional..."

but then say

"At the time of writing `Merriweather` only supports the..."

Should one of those be Merriweather Sans?

File: context/brand-context/springernature/README.md [hunk 5]
```diff
@@ -1,3 +1,89 @@
...
+- varLib instancer ([varLib instancer documentation](https://fonttools.readthedocs.io/en/latest/varLib/instancer.html))
+- woff2 library ([How To Convert Variable TTF Font Files to WOFF2](https://henry.codes/writing/how-to-convert-variable-ttf-font-files-to-woff2/))
+
+#### Generating Merriweather-Variable.woff2
+
+In the below commands, it is considered the CLI tools are in the `PATH`, if not
+you should use the absolute path to their binaries.
+
+1. Download Merriweather font under the name of `Merriweather-Variable.ttf`
+2. In the Terminal navigates to the folder you have just downloaded the font to
```
Comment (alexkilgour) [inline context/brand-context/springernature/README.md:48]: navigates -> navigate

(remove the S)

File: context/brand-context/springernature/README.md [hunk 6]
```diff
@@ -1,3 +1,89 @@
...
+5. Run `mv ./Merriweather-Variable-partial.woff2 ./Merriweather-Variable.woff2`
+6. Copy `Merriweather-Variable.woff2` to your app's font folder
+
+#### Generating MerriweatherSans-Variable.woff2
+
+In the below commands, it is considered the CLI tools are in the `PATH`, if not
+you should use the absolute path to their binaries.
+
+1. Download Merriweather Sans font under the name of `MerriweatherSans-Variable.ttf`
+2. In the Terminal navigates to the folder you have just downloaded the font to
```
Comment (alexkilgour) [inline context/brand-context/springernature/README.md:60]: navigates -> navigate

(remove the S)

File: context/brand-context/springernature/README.md [hunk 7]
```diff
@@ -1,3 +1,89 @@
+## Fonts
+
+We are using Merriweather as a variable font for both `serif` and `sans-serif`.
+
+Browsers who support variable fonts do support the `woff2` format. We therefore
+do not need to serve other formats.
+
+Browsers who do not support variable fonts, will fallback to the system fonts
```
Comment (jon-stevens) [inline context/brand-context/springernature/README.md:8]:
```suggestion
Browsers that do not support variable fonts will fallback to the system fonts
```

File: context/brand-context/springernature/README.md [hunk 8]
```diff
@@ -1,3 +1,89 @@
...
+specified in the `$context--font-family-serif` and `$context--font-family-sans`
+SASS variables after respectively `'Merriweather'` and `'Merriweather Sans'`.
+
+So far our `Merriweather` font instance only includes the [Weight variation
+axis](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Fonts/Variable_Fonts_Guide#weight).  
+At the time of writing `Merriweather` supports the additional [Width
```
Comment (jon-stevens) [inline context/brand-context/springernature/README.md:14]:
```suggestion
At the time of writing, `Merriweather` supports the additional [Width
```

File: context/brand-context/springernature/README.md [hunk 9]
```diff
@@ -1,3 +1,89 @@
...
+
+#### Tools needed
+
+- fonttools ([Install fonttools](https://pypi.org/project/fonttools/))
+- varLib instancer ([varLib instancer documentation](https://fonttools.readthedocs.io/en/latest/varLib/instancer.html))
+- woff2 library ([How To Convert Variable TTF Font Files to WOFF2](https://henry.codes/writing/how-to-convert-variable-ttf-font-files-to-woff2/))
+
+#### Generating Merriweather-Variable.woff2
+
+In the below commands, it is considered the CLI tools are in the `PATH`, if not
```
Comment (jon-stevens) [inline context/brand-context/springernature/README.md:44]:
```suggestion
In the following commands, it is assumed that the CLI tools are included in your `PATH`, if not
```

**Review by alexkilgour — APPROVED** [review]
👍

**Pull request merged**